src/Console/Command.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace DALTCORE\LaravelDeployHelper\Console;
+
+use Illuminate\Console\Command as BaseCommand;
+
+class Command extends BaseCommand
+{
+    /**
+     * @var string
+     */
+    protected $stage;
+
+    /**
+     * @var string
+     */
+    protected $branch;
+
+    /**
+     * Create a new command instance.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Setting stage and branch if they were given,
+     * else set them from the config if the indexes are found.
+     *
+     * If notting was found, the values will be null as aspected.
+     */
+    public function boot()
+    {
+        $config = config('laravel-deploy-helper');
+
+        $this->stage = $this->option('stage');
+        if (is_null($this->stage) && isset($config['default']['stage'])) {
```

Comment:
If you change it to `!is_null($this->stage)` it will try to override the `$this->stage` variable when it is already set from the command line using the flag.

That is why I check if `$this->stage` is null and if the default value is set on the config file, then set the `$this->stage` variable to the config setting